lib/fetch/sanitize.js

```diff
@@ -164,22 +168,46 @@ export function sanitizeBadger(json) {
     lastName: name && name.split(' ').slice(-1).join(' '),
     order: get('badger.order'),
     jobTitle: get('badger.job-title'),
-    primaryImageUrl: primaryImageUrl && primaryImageUrl.replace(/^http:/, 'https:'),
-    secondaryImageUrl: secondaryImageUrl && secondaryImageUrl.replace(/^http:/, 'https:'),
+    primaryImageUrl: primaryImageUrl &&
```

Comment:
This PR should just have added, it shouldn't have changed existing code. Alex's auto formatting makes it look worse than it is